docs/src/tutorials/anomaly_detection_unsupervised.ipynb

```diff
@@ -344,2008 +150,37 @@
     "    df[\"timestamp\"] = range(df.shape[0])\n",
     "    print(f\"found {df.shape[0]} events\")\n",
     "\n",
-    "    dataframes.append(df)\n",
+    "    raw_dataset_df.append(df)\n",
     "\n",
-    "dataframes[0].head(3)"
+    "raw_dataset_df[0].head(3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b9f50a64-c27e-4cda-afd9-f5da6dcc0276",
+   "metadata": {},
+   "source": [
+    "Next, we convert the dataset into a Temporian `EventSet`. Temporian will help data visualization and feature computation.\n",
+    "\n",
+    "**Note:** Our anomaly detection model can operate directly on raw time series data, and that's how we'll begin. However, preprocessing and augmenting the temporal data (for example, calculating moving statistics or calendar events) before applying the anomaly detection model can significantly boost its performance. For example of feature engineering, check the [Feature engineering section](https://temporian.readthedocs.io/en/latest/tutorials/m5_competition/#feature-engineering) in the M5 Competition tutorial.\n"
```

Comment:
For example of feature engineering --> For an example of feature engineering

Reply:
*Replaced